cmd/call-api-endpoint/main_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+)
+
+// Use the common main for all tests in this package.
+func TestMain(suite *testing.M) {
+	cmd.CMDServerTestingMain(suite)
+}
+
+// Test to ensure an API endpoint that requires parameters works correctly.
```

Comment:
I don't like this comment.
It's too conclusive (these tests are not conclusive).
I would just drop it, since this test is not tricky or special.